Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/ReadsOnMethods.dfy.expect

```diff
@@ -16,7 +16,6 @@ ReadsOnMethods.dfy(183,25): Error: insufficient reads clause to read field
 ReadsOnMethods.dfy(245,9): Error: insufficient reads clause to read field
 ReadsOnMethods.dfy(304,33): Error: insufficient reads clause to read field
 ReadsOnMethods.dfy(308,22): Error: insufficient reads clause to invoke function
-ReadsOnMethods.dfy(313,13): Error: call might violate context's modifies clause
```

Comment:
I wouldn't have expected that this error would go away. Can you explain a bit?

Reply:
Ultimately, the issue is that Boogie isn't guaranteed to report all errors in a given procedure. If it determines that some code is dead code (because a previous assertion was equivalent, in the current context, to `assert false`, for example), it may not even generate verification conditions for that part of the code.

Reply:
I checked with @robin-aws, who wrote this test, and this error wasn't an essential one for the test to cover. There's another test, for the concurrent attribute, that I refactored so that the two different errors occurred in different definitions and were therefore guaranteed to be reported.